src/SourceBuild/content/build.proj

```diff
@@ -28,6 +28,19 @@
   <Import Project="$(RepositoryEngineeringDir)build.sourcebuild.targets" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)sdkArchiveDiff.targets" Condition="'$(ShortStack)' != 'true' and '$(PortableBuild)' == 'true' and '$(PgoInstrument)' != 'true'" />
 
+  <!-- Copy SDK archive to assets root. -->
+  <Target Name="CopySdkArchive"
+          AfterTargets="Build">
+    <ItemGroup>
+      <_SdkArchive Include="$(ArtifactsAssetsDir)Sdk/**/dotnet-sdk-*$(ArchiveExtension)" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(_SdkArchive)"
+          DestinationFolder="$(ArtifactsAssetsDir)"
+          SkipUnchangedFiles="true"
+          UseHardlinksIfPossible="true" />
+  </Target>
```

Comment:
I thought more about this and realized that this target is a no-op for short stack builds. Please move this into installer.proj which is the unit that creates the archive.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9c2058b933a4dac3610accdcceff226d11565198